src/pages/transactions/common/validation/ValidationInterface.tsx

```diff
@@ -9,10 +9,10 @@
  */
 
 export interface TransactionValidation {
-  base_type: string;
-  date: string;
-  amount: string;
-  currency_id: string;
-  bank_integration_id: string;
-  description: string;
+  base_type?: string;
+  date?: string;
+  amount?: string;
+  currency_id?: string;
+  bank_integration_id?: string;
+  description?: string;
```

Comment:
The reason for that is this block of code, 

```
setErrors((prevState) => ({
        ...prevState,
        bank_integration_id: t('required_field'),
 }));
```

If I don't keep them optional, I will have to enter all of them but I only want to enter bank_integration_id. The reason I set up manual validation on the frontend is because the API returns a 500 error without any validation message instead of a 422 and with a validation message for this field, so I had to do it manually.